packages/presentation_layer/lib/app/screens/palindrome/palindrome.dart

```diff
@@ -29,7 +30,9 @@ class Palindrome extends BlocStateless<PalindromeBloc> {
                   padding: const EdgeInsets.symmetric(horizontal: 50),
                   child: TextField(onChanged: bloc.updateCurrentInputValue),
                 ),
-                Text(verifyResult),
+                screenLoadStatus!
```

Comment:
not the best way to force unwrap variables. use it in the following way

screenLoadStatus == true

Reply:
corrected